### PR TITLE
improved ActionView flows.rb documention to make it clearer [ci skip]

### PR DESCRIPTION
**Pull request by alexdotsh**

### CODE REVIEW

File: actionview/lib/action_view/flows.rb [hunk 1]
```diff
@@ -37,9 +37,8 @@ def initialize(view, fiber)
     end
 
     # Try to get stored content. If the content
-    # is not available and we are inside the layout
-    # fiber, we set that we are waiting for the given
-    # key and yield.
+    # is not available and we're inside the layout fiber
+    # then it will begin waiting for the given key and yield.
```
Comment (vipulnsward) [inline actionview/lib/action_view/flows.rb]: ..fiber, then..

File: actionview/lib/action_view/flows.rb [hunk 2]
```diff
@@ -60,8 +59,8 @@ def get(key)
     end
 
     # Appends the contents for the given key. This is called
-    # by provides and resumes back to the fiber if it is
-    # the key it is waiting for.
+    # by providing and resuming back to the fiber
```
Comment (vipulnsward) [inline actionview/lib/action_view/flows.rb]: ..fiber, if..

**Comment by vipulnsward** [comment]
Small changes and ready to merge. 
@alexmirk please add `[ci skip]` to commit message so that we don't run tests for this change on Travis.

**Comment by alexmirk** [comment]
Thanks @vipulnsward I have updated the commit message with [ci skip]

**Comment by vipulnsward** [comment]
@alexmirk please squash the commits.

File: actionview/lib/action_view/flows.rb (hunk 1 above)
Comment (vipulnsward) [inline actionview/lib/action_view/flows.rb]: @alexmirk as mentioned we need comma after fiber.

File: actionview/lib/action_view/flows.rb (hunk 2 above)
Comment (vipulnsward) [inline actionview/lib/action_view/flows.rb]: same here, we need comma.

**Pull request closed without merging**